.github/workflows/ci.yaml

```diff
@@ -17,7 +18,8 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macOS-latest]
-        rust: [stable]
+        rust: [nightly]
```

Comment:
While we don't need to double the number of runs by using `stable, nightly` here, I think we do need to test on stable. Perhaps test all backends on stable, but only on one OS?